srcpkgs/evolution-data-server/template

```diff
@@ -16,7 +16,7 @@ license="LGPL-2.1-only"
 homepage="https://wiki.gnome.org/Apps/Evolution"
 changelog="https://raw.githubusercontent.com/GNOME/evolution-data-server/gnome-3-30/NEWS"
 distfiles="${GNOME_SITE}/${pkgname}/${version%.*}/${pkgname}-${version}.tar.xz"
-checksum=de9cc30bdfaeff9dea561d7d1d43f31dfd24d0fdf3c231cfd3a9fc4f00e514c9
+checksum=91d6312f60752bb5229e35975b3c1f30f8f64bd8e5cce76cc275b1073d1725e9
 
 build_options="gir"
```

Comment:
If possible enable gir for this package :D

Reply:
Will do

Reply:
Alright, this might take a little longer, how about merging this now and enabling GIR sometime later?